scripts/src/chartprreview/chartprreview.py

```diff
@@ -181,13 +196,7 @@ def generate_verify_report(category, organization, chart, version):
             print("Either chart source or tar ball should exist.")
             sys.exit(1)
     if src_exists:
-        out = subprocess.run(["helm", "package", src], capture_output=True)
-        stdout = out.stdout.decode("utf-8")
-        print(stdout)
-        print(out.stderr.decode("utf-8"))
-        dn = os.path.dirname(stdout.split(":")[1].strip())
-        fn = os.path.basename(stdout.split(":")[1].strip())
-        out = subprocess.run(["docker", "run", "-v", dn+":/charts:z", "--rm", "quay.io/redhat-certification/chart-verifier:latest", "verify", os.path.join("/charts/", fn)], capture_output=True)
+        out = subprocess.run(["docker", "run", "-v", src+":/charts:z", "--rm", "quay.io/redhat-certification/chart-verifier:latest", "verify", "/charts"], capture_output=True)
```

Comment:
@mmulholla This is the line where the report is generated.

Reply:
but this line runs all checks - we thought only to run only digest computation.

Reply:
If I understand the code correctly it generates a report whether not a report file was submitted, if a report was not submitted it serves as the invocation of the report, if not it is just used to check the sha.

The problem with this approach is we are going to install and test the chart even when we only want the report for the sha value.  

Also if we generate a report when one was not submitted it looks as though we don' save it anywhere, but should we not include it in the PR?

Reply:
Just created a PR for my bash script. It includes an option  "checkdigest". More info available in the PR: https://github.com/openshift-helm-charts/repo/pull/63. Up to you whether you sue it or not.

Reply:
@pedjak I have updated the command to only invoke "-e has-readme" check.

@mmulholla The "report.yaml" is saved in the current directory. And later it is used for matching name and the version is in the same as in the directory structure, submitted report and the chart (Chart.yaml) I would prefer not to invoke the docker command multiple times.  When the user submits a chart without a report, the same functions are reused.

Reply:
Need to find some python lessons, sorry, now not sure how the second run does not overwrite the submitted one, just assure me that it will not.

Reply:
That run is happening within an "if" condition like this: `if os.path.exists(report_path):` That condition becomes true when a user submits the report and the else is executed otherwise.

Reply:
@mmulholla It will not overwrite the submitted one. The submitted report is in another directory.